AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockAssertUserNotLoggedActionGroup.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAdobeStockAssertUserNotLoggedActionGroup">
+        <waitForPageLoad stepKey="waitForLoad"/>
+        <dontSeeElement selector="{{AdobeStockSection.userImageSmall}}" stepKey="seeUserIcon" />
```

Comment:
Can you please add a check for an element that is visible on the page. For example "Sign In" link

Reply:
ok